### PR TITLE
Portfolio-setup-and-mobile-first

### PR DESCRIPTION
**Pull request by maximoortelli**
**This is my second attempt, do the project again. In this project, I implemented the following changes effectively:**

1. Index.html and style.css file fixed bugs reported to me by pros.
2. Correctly configure linters.
3. Added README documentation appropriately.
4. Links that redirect to my profiles were added.
5. I have used all the images from the figma template.
6. Added node_modules and test.md in .gitignore
7. MIT license added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            some of my
+            work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+            me.
+        </p>
+
+        <div class="lets-connect margin flex-column">
+
+            <p>LET'S CONNECT</p>
+
```
Comment (abdulhamiid) [inline index.html:43]: Kindly use a header tag hx (examples: `<h2>`, `<h3>`, `<h4>`, etc...) for the subtitle "LET'S CONNECT"

* This way will be semantically correct and the screen readers and other technologies will thank you. 💪😎

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @maximoortelli  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub flow

## Required Changes ♻️
_Please kindly check the comments under the review._

I'm unable to make an inline comment on your readme file, so please kindly check my comment below 👇 🤗

[REQUIRED] - Your license link in your readme file is broken, please kindly fix it, let's avoid broken links in our readme file for a professional repository. 🤗 🛠💪

image

image

Public repositories on GitHub are often used to share open-source software. For your repository to truly be open source, you'll need to license it so that others are free to use, change, and distribute the software. 👍


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (maximoortelli) [reply index.html:43]: Thank you very much abdulhamiid, I have already corrected everything mentioned.

Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @maximoortelli  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**